tracer/src/Datadog.Tracer.Native/dd_profiler_constants.h

```diff
@@ -57,15 +57,16 @@ const shared::WSTRING include_assemblies[]{
     WStr("Microsoft.Extensions.Logging"),
     WStr("Microsoft.Extensions.Logging.Abstractions"),
     WStr("System.Diagnostics.Process"),
-    WStr("Microsoft.Extensions.Identity.Core")
+    WStr("Microsoft.Extensions.Identity.Core"),
+    WStr("System.Runtime.Remoting"),
 };
 
 // Note that this list should be kept in sync with the values in shared/src/Datadog.Trace.ClrProfiler.Native/util.h
 const shared::WSTRING default_exclude_assemblies[]{
     WStr("dd-trace"),
     WStr("dd-trace.exe"),
     WStr("aspnet_state.exe"),
-    WStr("sqlservr.exe"),
+    WStr("sqlservr.exe")
```

Comment:
\*Shakes fist at missing trailing comma\* 😉

Reply:
Fixed in 424990f39cd3c504c09b7978f938c653e225a365